### ISSUE TITLE
BeastBot as LottoBot

### ISSUE THREAD

**Issue by kulverstukas1** [EvilzoneLabs/BeastBot#15]
Eventually need to implement LottoBot as a separate instance of BeastBot for reliability. So probably will have to fork the project and only make it do lotto, because it'll be a heavy constant-working bot. However this is just still a thought and many things haven't been discussed, such as prizes and how it would actually be played, but the initial idea is to have a simple ball-rolling bot at one day of the week where all purchased tickets would be in the game. In a nutshell: people buy tickets with numbers (using cookies or something), the bot has a date and time when rolls the balls, and you know the rest :P
http://en.wikipedia.org/wiki/Lottery

**Comment by Spacecow99** [comment]
The beauty of having a modular bot is you can just cut out the modules you don't need and write new ones for whatever you want it to do. It would require running a separate instance of the bot but no forking required :D

**Comment by kulverstukas1** [comment]
Yes, but I was gonna have a separate instance of the bot without any other modules that might interfere with the lotto and eliminate the possibility of crashing this bot when some other module lags up. If it's possible to have BeastBot modules run constantly with timed events firing on a set date, then I'll make a module - if not then I'll have to either implement that functionality or fork the bot and code it so it does only lotto and doesn't accept modules. Latter sounds easier :P
